test/blackbox-tests/test-cases/pkg/dev-tool-conflict-test.t

```diff
@@ -0,0 +1,256 @@
+Testing the conflicts that could occurs between the dependencies of dune-projects
+and dev-tools dependencies.
+
+The scenario here is, the fake OCamlForamt dev-tool depends on
+printer.1.0 and the project depends on a different version, printer.2.0.
+It shows those 2 does not conflicts and the dev tools dependencies do not leak
+into the user build environment.
+
+  $ . ./helpers.sh
+  $ mkrepo
+
+Enable the feature
+  $ DUNE_CONFIG__LOCK_DEV_TOOL=enabled
+
+Make a fake OCamlFormat:
```

Comment:
Are these changes correct to format OCamlFormat as the proper name, or should it be `ocamlformat` because like in the code example?

Reply:
In this case, OCamlFormat is the correct way.

Reply:
Thanks!